### PR TITLE
add Ayushparikh-code

### LINKED ISSUE #519: Ayush Parikh
githubprofile

### PR DESCRIPTION
**Pull request by Ayushparikh-code**
resolve #519

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "name": "Ayush Parikh",
+    "githubUsername": "Ayushparikh-code",
+    "imageUrl": "https://user-images.githubusercontent.com/60268067/118812871-33649100-b8cc-11eb-9e90-0ec4fe333229.png",
+    "issueNumber": 519
+}
```
Comment (starlightknown) [inline data.json:7]:
```suggestion
  },
```

File: data.json [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+
```
Comment (starlightknown) [inline data.json:8]:
```suggestion
```

**Review by starlightknown — CHANGES_REQUESTED** [review]
Also could you add your name in the last?

**Comment by schmelto** [comment]
With the suggestions from @starlightknown and moving down to the last its looking good for me

**Review by schmelto — CHANGES_REQUESTED** [review]
With the suggestions from @starlightknown and moving down to the last its looking good for me

**Review by Ayushparikh-code — COMMENTED** [review]
done

**Comment by schmelto** [comment]
@Ayushparikh-code could you add yourself at the bottom of the data.json

**Comment by Ayushparikh-code** [comment]
why checks are failing ?

**Comment by Ayushparikh-code** [comment]
i already added myself

**Comment by Ayushparikh-code** [comment]
can u review again !

**Comment by Ayushparikh-code** [comment]
@starlightknown @schmelto ?

**Comment by starlightknown** [comment]
> i already added myself

Add yourself in the last please.

**Comment by Ayushparikh-code** [comment]
> > i already added myself
> 
> Add yourself in the last please.

i did

**Comment by Ayushparikh-code** [comment]
@can u plz fork my repo and do this for me I don't understand why checks are failing

**Review by Ayushparikh-code — COMMENTED** [review]
done

**Review by starlightknown — APPROVED** [review]
Looks good now

**Comment by Ayushparikh-code** [comment]
> Looks good now

yes

**Review by Ayushparikh-code — COMMENTED** [review]
done

**Comment by Ayushparikh-code** [comment]
> Looks good now

what does it mean ? am getting this -  Error: A branch or tag with the name 'patch-1' could not be found

**Comment by schmelto** [comment]
#521

**Pull request closed without merging**